finance-api/concepts/how-card-payments-work-at-Zettle.md

```diff
@@ -0,0 +1,21 @@
+How card payments work at Zettle
+===
+For every merchant who makes transactions with Zettle, they have an Zettle account that handles those transactions.
+
+As shown in [Figure 1 Zettle card transaction flow](#Zettle-card-transaction-flow), after a merchant makes a transaction with Zettle, the money of the transaction will go through the following phases in sequence:
+1. Recorded in the merchant's Zettle preliminary account:
+ 
+   Zettle checks the money with a third-party acquirer who confirms the transaction.
```

Comment:
What is third-party acquirer? do we need to mention that? 
Shouldn't we just write "acquiring bank"?